Source/ENS/Private/Characters/Player/EnsPlayerController.cpp

```diff
@@ -22,6 +25,8 @@ AEnsPlayerController::AEnsPlayerController()
 void AEnsPlayerController::BeginPlay()
 {
 	Super::BeginPlay();
+	ActiveCamera = Cast<UCameraComponent>(GetPawn()->GetComponentByClass(UCameraComponent::StaticClass()));
+	ActiveCapsuleComponent = Cast<UCapsuleComponent>(GetPawn()->GetComponentByClass(UCapsuleComponent::StaticClass()));
```

Comment:
maybe the same here, evaluate if we have an easy access to it. If this is only one call, we may remove these variables completly.